src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const buttons = document.querySelectorAll('.button');
+const tbody = document.querySelector('tbody');
+const minLimit = 2;
+const maxLimit = 10;
+let countRow = tbody.rows.length;
+let countColumn = tbody.rows[0].cells.length;
+
+document.addEventListener('click', events => {
+  const target = events.target;
+
+  if (!target.classList.contains('button')) {
+    return;
+  }
+
+  if (target.classList.contains('append-row')) {
+    const tr = document.createElement('tr');
```

Comment:
try to use more meaningful variable  E.g. `tableRow`. Only 2 symbols may confuse  with function calls and other operations with variable